src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -465,6 +512,51 @@ public virtual KeyedHashAlgorithm CreateKeyedHashAlgorithm(byte[] keyBytes, stri
             }
         }
 
+        private RsaKeyWrapProvider CreateRsaKeyWrapProvider(SecurityKey key, string algorithm, bool willDecrypt)
```

Comment:
The operations on KeyWrap are Wrap and UnWrap. the parameter named willDecrypt seems incorrect.